_posts/2016-09-22-ignobel.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Want to win an Ig Nobel? Here are some suggestions.
+author: neuroamanda
+---
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/matthewcobb">@matthewcobb</a> <a href="https://twitter.com/ProfBrianCox">@ProfBrianCox</a> got to be worth an Ignobel...??</p>&mdash; Russell Blinman (@Georgiasdad1) <a href="https://twitter.com/Georgiasdad1/status/771659917308547072">September 2, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Science writers are always trying to think of fascinating story ideas that can be backed up with research.
+Inspiration might spring from the most unlikely of sources, like a nonsense musing on _Seinfeld_ or the historical minutiae of a Trivial Pursuit question.
+The hard part is finding research about these obscure, fleeting puzzlings. My quest often goes unfulfilled.
+
+Luckily, there is a prize that motivates researchers to conduct just these kinds of investigations.
+For figuring out things like [why woodpeckers don't get headaches](http://www.ucdmc.ucdavis.edu/welcome/features/20061011_ig_nobel_schwab/index.html), or how to [un-boil an egg](https://dx.doi.org/10.1002%2Fcbic.201402427), or [whether banana skins actually make you slip](https://dx.doi.org/10.2474%2Ftrol.7.147), intrepid scientists are awarded the [Ig Nobel Prize](http://www.improbable.com/ig/), the down-to-earth and tongue-in-cheek Nobel Prize spoof that is surprisingly revealing about what counts as relatable science and how the scientific method can be brought to bear on the humorous, the smutty, and the bizarre.
+(The 2016 Ig Nobel ceremony on September 22 is being [livestreamed here](http://www.improbable.com/ig/2016/).)
+
+In no particular order, here are some Ig Nobel-worthy contenders, as proposed by the Twitterverse:
+* [When zombies attack: mathematical modelling of an outbreak of zombie infection](https://www.math.upenn.edu/~ted/203S10/Projects/Zombies/Zombies.pdf) (Related: [A mathematical model of Bieber Fever: the most infectious disease of our time?](http://mysite.science.uottawa.ca/rsmith43/BieberFever.pdf))
```

Comment:
When hotlinking to a PDF, could you include "[PDF]" at the end of the text as part of the hyperlink? It's good for those who are on phones and can't see by hovering where the link will take them.